### ISSUE TITLE
Titler på højskolesange

### ISSUE THREAD

**Issue by jvenborg** [jvenborg/unghanekorets-sangsamling#1]
Det ligner umiddelbart, at du under `layouts/_default/single.html` har ændret titlen til at blive:

### Titel
**Nummer (18. udgave) 320
Nummer (19. udgave): 321**

frem for:
### 321. Titel
**Nummer (18. udgave): 320**

Er det mon korrekt forstået? Kan jo desværre ikke lige smække op på hjemmesiden ;) Men gør jeg snarest. Det er selvfølgelig en fin løsning, omend jeg godt gad at have nummeret inkluderet i titlen! Som måske er lidt mere rent på en måde.

**Comment by Mads-G-H** [comment]
Ja, det er rigtigt forstået, men jeg har egentlig ikke noget imod at ændre det tilbage til det "gamle" format. Min primære indvending mod det "gamle" format er, at det ikke er helt klart, hvad der skal ske, hvis vi har en sang i samlingen, der bliver taget ud af en kommende udgave af Højskolesangbogen. Desuden er det måske ikke alle brugere, der er helt skarpe på, hvilken udgave nummeret refererer til, hvis man skriver "321. Titel".
